### ISSUE TITLE
Building from source

### ISSUE THREAD

**Issue by NospamNospam** [Kevga/BeatSaberCinema#78]
Hello. Can you please list the prerequisites for building the project from source, VS loadouts included?

**Comment by Kevga** [comment]
See the "Contributing" section of the readme.

**Comment by NospamNospam** [comment]
Yes, I added the user config, but some references are still unresolved HMUI for example.

**Comment by Kevga** [comment]
There is nothing else. If HMUI, which is part of the game, remains unresolved, then you likely didn't set up the user config correctly.

**Comment by NospamNospam** [comment]
And what would be incorrect if the config points to the latest version installed? Actually, the project builds now, after adding some mods to BS, despite the dependencies still being partially unresolved. Maybe some dependencies are missing from the readme? And I'm using Visual Studio CE, if it has anything to do with this.

**Comment by thisrabbit** [comment]
Please check whether your game version matches the mod repository version, and this is how I resolve the failed-to-build error even if I just cloned this repo without any modification. As you can see, every release of the game itself changes its implementation details and mods have to adjust accordingly, and if they are not matched, errors raise.
